doc/DefaultAnalytics.md

```diff
@@ -0,0 +1,26 @@
+# Default Analytics
+
```

Comment:
Should we use `ObjectType` on all our types? Makes it nicer imo.

Reply:
allows to customize the default behavior or all DataLoaders. => allows to customize the default behavior or all `DataLoaders`.

Reply:
`DataLoaders` isn't a type; `DataLoader` is.

Reply:
Ohh I misread.